<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DVD List Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="genreLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genreEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>292</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="locationLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="notesEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>791</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="NotesOrSearch">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lockEntry">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DVD List Query Form</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="modeIndicator">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Search Mode</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="searchResults">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>991</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLocation">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>----</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Summer House</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Living Room</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Small Bedroom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Large Bedroom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>51 Waddington Ave</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="hint">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hint: Use % as a wildcard, e.g. 'Return%King'</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelID">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>70</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxID">
    <property name="geometry">
     <rect>
      <x>626</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteID">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>410</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete by ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteTitle">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>410</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete by Title</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSQL_Logs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSQL_Logs">
   <property name="text">
    <string>SQL Logs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
